### DISCUSSION TITLE
generated obsidian URI doesnt work if Vault Name contains spaces

### DISCUSSION THREAD

**Discussion by Z3krom991** [Taitava/obsidian-shellcommands#412] [Debugging and testing]
### Obsidian debug information

Obsidian version: v1.5.12
Operating system name and version: Windows 10 Pro 22H2
Shell commands version: 0.22.0

### What shell are you using?

CMD.EXE

### Please describe the problem

When the Name of my vault contains spaces, the URI opens another obsidian instance, with the same specific file open each time.

Shell Command: `start C:\\"Program Files"\\Adobe\\"Acrobat DC"\\Acrobat\\Acrobat.exe /A "page=94" C:\\Users\\USER\\Documents\\PnP-Content\\Regelwerke\\Mondstahlklingen.pdf`

Generated URI: obsidian://shell-commands/?vault=Gefangen%20im%20Labyrinth&execute=hx1e1oh6uy

After renaming the Vault, removing the spaces, the newly generated URI worked without issues. Shell Command stayed the same.

New Generated URI: obsidian://shell-commands/?vault=Gefangen_im_Labyrinth&execute=hx1e1oh6uy

**Comment by Taitava** [comment]
Hi. Where do you open the URI link from? A web browser? Obsidian?

This seems to be a strange problem. I have vaults with spaces in their names and for me the URIs with `%20` work fine on Windows 10.

Just to narrow down the problem, does this URI work for you: `obsidian://shell-commands/?vault=Gefangen+im+Labyrinth&execute=hx1e1oh6uy`
I have replaced `%20` with `+`, which should also produce spaces. Edit: Actually, the `+` version does not work for me. Obsidian can't find the vault now, so I assume it won't work for you either.
